descartes_planner/include/descartes_planner/planning_graph.h

```diff
@@ -144,17 +146,17 @@ class PlanningGraph
   //       and include an accessor to both formats
 
   // maintains the original (Cartesian) points list along with link information and associated joint trajectories per point
-  std::map<descartes_core::TrajectoryPt::ID, CartesianPointInformation> *cartesian_point_link_;
+  CartesianMap* cartesian_point_link_;
```

Comment:
This might be a good opportunity to remove the pointer syntax from this member unless you think it needs to stay a pointer.